MIPTools.def

```diff
@@ -189,6 +189,10 @@ From: amd64/ubuntu:20.04
     # install parasight
     scp /opt/programs/parasight_v7.6/parasight.pl /opt/bin/parasight76.pl
 
+    # install basespace cli
+    BS_PATH="https://launch.basespace.illumina.com/CLI/latest/amd64-linux/bs"
```

Comment:
Instead of downloading the latest version of the CLI tool, we can specify the version number by replacing `latest` with a version number. The available versions can be found [here](https://launch.basespace.illumina.com/CLI/latest/latest.html).